backend/tournesol/views/exports.py

```diff
@@ -12,12 +12,33 @@
 
 from core.models import User
 from tournesol.entities.base import UID_DELIMITER
-from tournesol.lib.public_dataset import get_dataset
+from tournesol.lib.public_dataset import get_dataset, get_user_dataset
 from tournesol.models import Comparison, Poll
 from tournesol.serializers.comparison import ComparisonSerializer
 from tournesol.utils.cache import cache_page_no_i18n
 from tournesol.views.mixins.poll import PollScopedViewMixin
 
+README_CONTENT = """\
+This is the public database of Tournesol (https://tournesol.app/).
```

Comment:
I made few little suggestions about the content of `README_CONTENT` here: https://github.com/tournesol-app/tournesol/pull/1154#issuecomment-1277681110